source/dub/commandline.d

```diff
@@ -470,6 +811,32 @@ struct CommandGroup {
 	}
 }
 
+/******************************************************************************/
+/* HELP                                                                       */
+/******************************************************************************/
+
+class HelpCommand : Command {
+
+	this()
+	{
+		this.name = "help";
+		this.description = "Shows the help message";
+		this.helpText = [
+			"Shows the help message and the supported command options."
+		];
+	}
+
+	/// There is nothing to prepare
+	override void prepare(scope CommandArgs args)
+	{
+		assert(false, "There is nothing to prepare");
```

Comment:
this will result in something like
```
core.exception.AssertError@commandline.d(832): There is nothing to prepare
----------------
??:? _d_assert_msg [0x559093d2d9ce]
commandline.d:832 void dub.commandline.HelpCommand.prepare() [0x559093d2cb18]
```

To someone not having seen this code this isn't very helpful. A more helpful message would be something like:

HelpCommand.prepare is not supposed to be called, use `cast(HelpCommand)this` to check if help was requested before execution.